amd/src/crossword_question.js

```diff
@@ -283,6 +283,10 @@ export class CrosswordQuestion {
                     const clueInputEl = this.options.crosswordEl
                         .closest('.qtype_crossword-grid-wrapper')
                         .querySelector(`.wrap-clue[data-questionid='${wordId}'] input`);
+                    // Replace spaces with an underscore character before binding to the answer input.
+                    if (key === " ") {
+                        key = '_';
+                    }
```

Comment:
Mixture of quotes is a bit sloppy. Can you consistently use ' ' unless you need " ".